rustfmt.toml

```diff
@@ -13,6 +13,12 @@ ignore = [
 
     # tests for now are not formatted, as they are sometimes pretty-printing constrained
     # (and generally rustfmt can move around comments in UI-testing incompatible ways)
+    "/tests/*",
+
+    # but we still want to format rmake.rs files in tests/run-make/ so we need to do this
+    # dance to avoid the parent directory from being excluded
+    "!/tests/run-make/",
+    "/tests/run-make/*/*.rs",
```

Comment:
There is no specific condition for rmake files as far as I see here. Does this actually work? And can't `!rmake.rs` just work?

Reply:
The condition is for them the same as below, it's one line below:

```
    "!/tests/run-make/*/rmake.rs",
```

Reply:
I wonder if

```
     "/tests/*",
     "!rmake.rs",
```

would work.

Reply:
I don't think it will, the `tests/run-make` directory will be ignored and so all it's subdirectories and files will also be ignored.

That's why I added the `!/tests/run-make/` exception.